### PR TITLE
Update docstrings to NumPy standard (`strategies/json_ld`)

### PR DESCRIPTION
**Pull request by Cerebrovinny**

### CODE REVIEW

**Comment by Cerebrovinny** [comment]
@michaelweinold please review this

File: bw2io/strategies/json_ld.py [hunk 1]
```diff
@@ -86,17 +240,57 @@ def json_ld_add_activity_unit(db):
...
-    """Change metadata field names from the JSON-LD `processes` to BW schema.
+    """
+    Change metadata field names in the given database to match the Brightway schema.
 
-    BW schema: https://wurst.readthedocs.io/#internal-data-format
+    Takes a database represented as a list of dictionaries and changes the metadata field names in the 
+    'processes' to match the Brightway schema. This is done by using a mapping between the old and new field names.
 
-    """
+    Brightway schema: https://wurst.readthedocs.io/#internal-data-format
```
Comment (michaelweinold) [inline bw2io/strategies/json_ld.py:275]: @cmutel - is this something you wanted to move to the main documentation? I remember you mentioned something along these lines.

Comment (cmutel) [reply bw2io/strategies/json_ld.py:275]: Sorry, this is incorrect, the `wurst` data format is slightly different. The Brightway schema is here: https://documentation.brightway.dev/en/latest/source/introduction/introduction.html#activity-data-format

Comment (michaelweinold) [reply bw2io/strategies/json_ld.py:275]: @Cerebrovinny - could you address this in your next PR? Thanks!

Comment (Cerebrovinny) [reply bw2io/strategies/json_ld.py:275]: sure I can cover that

**Pull request merged**